ogc/src/main/scala/geotrellis/server/ogc/wcs/CoverageView.scala

```diff
@@ -50,7 +55,9 @@ class CoverageView(
 }
 
 object CoverageView {
-  def sourceDescription(source: OgcSource): CoverageDescriptionType = {
+
+  def sourceDescription(sources: List[OgcSource]): CoverageDescriptionType = {
+    val source = sources.head
```

Comment:
Just calling out here that we're now explicitly using the first source in the list of valid sources for a given identifier to construct all of the spatial domain information for a layer. This was happening implicitly before because there was a 1 to 1 mapping of identifier to ogc source. With the addition of temporal slices, that's no longer the case. This assumption will continue to work as long as our list of sources all have the same spatial domain and CRS.